alpha-blog/app/models/user.rb

```diff
@@ -1,4 +1,7 @@
 class User < ApplicationRecord
-
+  has_many :articles
```

Comment:
Would all the articles of this user be deleted if this user is deleted?

Reply:
No. the `user_id` of the article will still be there. Just that it cant refer to the Users table because the data is not there, but the article data is not deleted. Therefore the user name will become blank

Reply:
Why would you want to keep the article if the user data is deleted?

Reply:
because the video has not reached the point where I can delete users so it is not implemented yet